core/src/main/scala/org/ensime/core/AnalyzerManager.scala

```diff
@@ -155,7 +105,7 @@ class AnalyzerManager(
     broadcaster ! Broadcaster.Persist(AnalyzerReadyEvent)
     broadcaster ! Broadcaster.Persist(FullTypeCheckCompleteEvent)
     // initialize loadedFiles
-    config.projects foreach (p => historyOfModule += (p -> new LoadedFilesData(p)))
+    config.projects foreach (p => historyOfModule += (p -> LoadedFilesData(Map(), NotLoaded)))
```

Comment:
`Map.empty` is slightly better performance than `Map()` because the latter will create a zero element sequence. Only usually relevant for code in a tight loop, but worth you knowing about.